gh-prs

```diff
@@ -66,12 +66,20 @@ if [ -n "$static" ]; then
   exit
 fi
 
-# Make sure fzf is available
-if ! type -p fzf >/dev/null; then
-  echo "fzf not found on the system" >&2
-  exit 1
+# Make sure fzf or sk is available
+declare -a selectors=("fzf" "sk")
+selector=""
+for s in "${selectors[@]}"; do
+  if type -p "$s" >/dev/null; then
+    selector=$s
+    break
+  fi
+done
+
+if [[ ! -n ${selector} ]]; then
+  echo "${selectors[@]} are not found on the system" >&2
 fi
```

Comment:
We should exit if a fuzzy finder isn't found. I'd also like to keep the error message the same to be a little more clear on when it fails.

There's also a couple things shellcheck reported.

```suggestion
if [ -z "${selector}" ]]; then
  echo "fzf not found on the system" >&2
  exit 1
fi
```